oled/error.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2016 Richard Hull and contributors
+# See LICENSE.rst for details.
```

Comment:
What do you think about adding a header like this instead of the big chunk of text? Other projects with the MIT license like Twisted do the same.